litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx

```diff
@@ -84,13 +89,52 @@ const WorkflowTable = forwardRef(({ isCustom }: WorkflowTableProps, ref) => {
     setExperiments(expData);
   };
 
+  // Revert Chaos
+  const toggleRevertChaos = (manifest: string) => {
+    let deleteEngines = 'kubectl delete chaosengine ';
+    const parsedYAML = YAML.parse(manifest);
+
+    parsedYAML.spec.templates[0].steps.push([
+      {
+        name: 'revert-chaos',
+        template: 'revert-chaos',
+      },
+    ]);
+
+    parsed(manifest).forEach((engine) => {
+      deleteEngines = `${deleteEngines + engine} `;
+    });
+
+    deleteEngines += '-n {{workflow.parameters.adminModeNamespace}}';
+
+    parsedYAML.spec.templates[parsedYAML.spec.templates.length] = {
+      name: 'revert-chaos',
+      container: {
+        image: 'litmuschaos/k8s:latest',
```

Comment:
this is a constant i guess which may be declared in some file or at the variable declaration part above. @S-ayanide

Reply:
This is a part of the YAML, thus better not to have it in constants.